### ISSUE TITLE
Optical coherence tomography with abnormal peripheral retina abnormality but macular sparing

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10600]
**Preferred term label:**
Optical coherence tomography with abnormal peripheral retina abnormality but macular sparing

**Synonyms**
Foveal sparing

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/browse/term/HP:0030603 as the grandparent term, may need Abnormal retinal morphology on macular OCT as a parent term.

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://www.ncbi.nlm.nih.gov/books/NBK549574/
"OCT findings in RPE65-LCA/EOSRD can be variable and can include a preserved central foveal region with thinning of the outer nuclear layer (ONL) that surrounds the fovea "

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
the way to code this in HPO would be to EXCLUDE Abnormal foveal morphology and to code the specific OCT finding. The proposed label is "bundled" and as such not a good HPO term...

**Comment by MickeySegal** [comment]
How about a modifier term "Optical coherence tomography macular sparing" ?
